lib/streamlit/connections/snowflake_connection.py

```diff
@@ -25,8 +25,11 @@
 from streamlit.connections import BaseConnection
 from streamlit.connections.util import running_in_sis
 from streamlit.errors import StreamlitAPIException
+from streamlit.logger import get_logger
 from streamlit.runtime.caching import cache_data
 
+LOGGER = get_logger(__name__)
```

Comment:
nit: `_LOGGER: Final = logger.get_logger(__name__)`  to follow most of the places we use the logger